### PR TITLE
Add 'standalone rtsan' header file

### PR DESCRIPTION
**Pull request by cjappl**

### CODE REVIEW

File: rtsan.h [hunk 1]
```diff
@@ -0,0 +1,129 @@
+//===-- rtsan.h -------------------------------------------------*- C++ -*-===//
```
Comment (cjappl) [inline rtsan.h:1]: First off, this is in the very top level of the repo, first level. Do we want to add it to a directory? `include` or `rtsan` or similar?

File: rtsan.h [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+#ifdef __cplusplus
+} // extern "C"
+
+namespace __rtsan {
+#if defined(__SANITIZE_REALTIME)
+
+inline void Initialize() { __rtsan_ensure_initialized(); }
+
+class ScopedRealtime {
```
Comment (cjappl) [inline rtsan.h:57]: Do we like this class name?

File: rtsan.h [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+// other rtsan functions are called.
+void __rtsan_ensure_initialized(void);
+
+bool __rtsan_is_initialized(void);
+
+#ifdef __cplusplus
+} // extern "C"
+
+namespace __rtsan {
+#if defined(__SANITIZE_REALTIME)
```
Comment (cjappl) [inline rtsan.h:53]: Do we like this being our "on/off switch"?

File: rtsan.h (hunk 2 above)
Comment (davidtrevelyan) [reply rtsan.h:57]: True. Let's discount `ScopedEnabler`. Something doesn't really feel right about `ScopedNonblocking`. 

How about `ScopedSanitizeRealtime`?

File: README.md [hunk 4]
```diff
@@ -146,9 +146,52 @@ RTSan's algorithm consists of two parts that work together:
 
 # Using RTSan with other compilers
 
-Yes, it's possible. Here's how.
+The recommended way to use RTSan is to use it with LLVM 20 directly, as described elsewhere in this document. The rest of this section describes a hack which may or may not continue to work in the future.
```
Comment (cjappl) [inline README.md:149]: This is an initial draft of these docs, I need to actually go through this example and make sure it works on both systems. It should be **mostly** right, but I haven't checked it out

**Review by davidtrevelyan — APPROVED** [review]

**Pull request merged**